sdk/keyvault/keyvault-secrets/src/index.ts

```diff
@@ -26,6 +26,7 @@ import "@azure/core-paging";
 import { PageSettings, PagedAsyncIterableIterator } from "@azure/core-paging";
 import {
   SecretBundle,
+	DeletedSecretBundle,
```

Comment:
Might want to align this with the other imports

Reply:
oh, let me pass prettier!